var/spack/repos/builtin/packages/py-backports-zoneinfo/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyBackportsZoneinfo(PythonPackage):
+    """Backport of the standard library zoneinfo module"""
+
+    homepage = "https://github.com/pganssle/zoneinfo"
+    pypi = "backports.zoneinfo/backports.zoneinfo-0.2.1.tar.gz"
+
+    version("0.2.1", sha256="fadbfe37f74051d024037f223b8e001611eac868b5c5b06144ef4d8b799862f2")
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-setuptools@40.8.0:", type="build")
+    depends_on("py-importlib-resources", type=("build", "run"), when="^python@:3.6")
```

Comment:
Spack does not support Python 3.6 and older, so you can just remove this line. You could also remove the `python@3.6:` constraint as well.